### DISCUSSION TITLE
Change ADH port?

### DISCUSSION THREAD

**Discussion by Fr3d3rick83** [AdguardTeam/AdGuardHome#5552] [Q&A]
Hi, I installed ADH on my Synology via Docker with port 8080. But now I need to change the port, how can I do this?

**Comment by ammnt** [comment]
@Fr3d3rick83, check this section in wiki:
https://github.com/AdguardTeam/AdGuardHome/wiki/Docker#create-and-run-the-container

You can just recreate the docker container with the another ports:
```
    -p 53:53/tcp -p 53:53/udp\
    -p 67:67/udp -p 68:68/udp\
    -p 80:80/tcp -p 443:443/tcp -p 443:443/udp -p 3000:3000/tcp\
    -p 853:853/tcp\
    -p 784:784/udp -p 853:853/udp -p 8853:8853/udp\
    -p 5443:5443/tcp -p 5443:5443/udp\
```
